mongodb/archivetracking.go

```diff
@@ -32,6 +34,7 @@ type trackedArchive struct {
 	DbToken    thread.Token `bson:"db_token"`
 	BucketKey  string       `bson:"bucket_key"`
 	BucketRoot []byte       `bson:"bucket_root"`
+	Owner      []byte       `bson:"owner"`
```

Comment:
New property to track the archive owner so we can get their ffs info while running the tracker